contentcuration/contentcuration/frontend/shared/leUtils/ContentKinds.js

```diff
@@ -26,3 +28,9 @@ export const ContentKindsNames = {
   VIDEO: 'video',
   ZIM: 'zim',
 };
+
+export const ContentKindLearningActivityDefaults = {
+  [ContentKindsNames.AUDIO]: LearningActivities.LISTEN,
```

Comment:
I think it would be good to replicate the full mapping we have on the backend, here?

https://github.com/learningequality/studio/blob/a642f2ad00e9b8188a26892e225faec966dc3d4e/contentcuration/contentcuration/constants/contentnode.py#L6

Reply:
I can do that. I only added the ones that were explicitly shown in the designs, given one shown in the design showed no default (.zip)

Reply:
Hrm, that's fair - maybe it's only the slideshow that should be added then.

Are we adding practice by default for exercises somewhere else?

Reply:
updated!